addons/repair/models/repair.py

```diff
@@ -49,8 +50,8 @@ class Repair(models.Model):
     state = fields.Selection([
         ('draft', 'Quotation'),
         ('confirmed', 'Confirmed'),
-        ('under_repair', 'Under Repair'),
         ('ready', 'Ready to Repair'),
+        ('under_repair', 'Under Repair'),
         ('2binvoiced', 'To be Invoiced'),
         ('invoice_except', 'Invoice Exception'),
```

Comment:
This state is never used (I think), can you remove it ? (not really related, but it painful to keep useless code for nothing)

Reply:
It's actually part of the normal flow, after you click "Start Repair".

Reply:
On the other hand, I see that 'invoice_except' is not used anywhere (maybe you wanted to mention this one), I'll remove it.